machine-learning/README.md

```diff
@@ -53,15 +53,19 @@ $ ./manage-ml.js list
 fish_detector    8716935   vision
 barcode_scanner  8716959   vision
 smart_reply      8716981   natural_language
-$ ./manage-ml.js new ~/yak.tflite yak_detector --tags vision,experimental
+$ ./manage-ml.js new yak_detector -f ~/yak.tflite --tags vision,experimental
 Uploading model to Cloud Storage...
 Model uploaded and published:
 yak_detector     8717019   experimental, vision
+$ ./manage-ml.js new emu_detector -a projects/12345/locations/us-central1/models/ICN12345
+Model uploaded and published:
+emu_detector     8717033
 $ ./manage-ml.js update 8717019 --remove_tags experimental
 $ ./manage-ml.js delete 8716959
 $ ./manage-ml.js list
 fish_detector    8716935   vision
 smart_reply      8716981   natural_language
```

Comment:
This one confused me temporarily. At first I thought you were implying we support AutoML natural language models, but we don't. Can we make it more clear that it's just a custom model that the user came up with and has nothing to do with AutoML?

Reply:
This example isn't specifically about AutoML -- it's an example transcript of someone using the sample. I just added a new part about AutoML. The smart_reply/natural_language part was already there.

Reply:
Yes. It was fine when everything was about custom models. But now that we've added AutoML models (which do have natural language options) it might be confusing since we don't support the natural language options... Only vision. So it might be best to remove the references to natural language or somehow make it obvious that it's a custom model, not an autoML model. Otherwise I can see people trying to use natural language autoML models with Firebase and failing because they are not supported.